drccobalt/css/style.css

```
@@ -0,0 +1,216 @@
+.body {
+    margin: 0;
+    border-width: 0;
+    box-sizing: border-box;
+    height: 100vh;
+    scroll-behavior: smooth;
+  }
+
+
+  .title-container {
+    height: 100vh;
+    width: 100vw;
```

Your page is "overflowing" (i.e., making it so that a horizontal scrollbar is necessary) because your `.title-container`, `.cobalt-info`, and `.map-container` elements are all set to `100vw`, and when added to the default margin around a `body` element the total width works out more more than 100 viewport units. I recommend either:
1. changing the width in these three rules from `100vw` to `100%`, or
2. adding a rule to get rid of the space around the body (`body { margin: 0; padding: 0; }`)

```suggestion
    width: 100%;
```